earthpy/io.py

```diff
@@ -91,6 +93,9 @@
 DATA_NAME = op.join("earth-analytics", "data")
 ALLOWED_FILE_TYPES = ["file", "tar", "tar.gz", "zip"]
 
+# Create logger object
+logger = logging.getLogger(__name__)
```

Comment:
here we create the logger -- this works but this logger is at the default LEVEL so it does not print.